templates/git_client_side_hooks/pre-push

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
```

Comment:
I don't think we need a new top-level directory for this. I think it can go under `hack/git_client_side_hooks`